### PR TITLE
DOCS-1871 Load Balancer Rate Limits

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: runtime-manager/v/latest/lb-create-api.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= To Create a Load Balancer using the API
+= To Create a Dedicated Load Balancer using the API
 
 You can create an external load balancer using the CloudHub API.
```
Comment (sundippatel) [inline runtime-manager/v/latest/lb-create-api.adoc]: should we swap "external" with "dedicated"?

File: runtime-manager/v/latest/lb-create-arm.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= To Create a Load Balancer using Runtime Manager
+= To Create a Dedicated Load Balancer using Runtime Manager
 
 You can use Runtime Manager to create load balancer in your CloudHub environment.
```
Comment (sundippatel) [inline runtime-manager/v/latest/lb-create-arm.adoc]: Suggestion: 
"..to create a dedicated load balancer.."

File: runtime-manager/v/latest/lb-create-api.adoc (hunk 1 above)
Comment (sleslie23) [reply runtime-manager/v/latest/lb-create-api.adoc]: @sundippatel Yes!

File: runtime-manager/v/latest/lb-create-arm.adoc (hunk 2 above)
Comment (sleslie23) [reply runtime-manager/v/latest/lb-create-arm.adoc]: Will fix!

File: runtime-manager/v/latest/lb-create-cli.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= To Create a Load Balancer using the CLI
+= To Create a Dedicated Load Balancer using the CLI
 
 You can create an external load balancer using the Anypoint Platform CLI.
```
Comment (sundippatel) [inline runtime-manager/v/latest/lb-create-cli.adoc]: should we also swap "external" with "dedicated"?

Comment (sleslie23) [reply runtime-manager/v/latest/lb-create-cli.adoc]: Yes! Will fix.

Comment (sleslie23) [reply runtime-manager/v/latest/lb-create-cli.adoc]: Fixed.

**Comment by sleslie23** [comment]
Received confirmation from Federico Bongiovanni that " the load balancer returns a 503 Service Unavailable response" when the rate limit is reached.

**Pull request merged**